api/src/main/java/org/openmrs/module/dhisreporting/Configurations.java

```diff
@@ -19,6 +19,10 @@ private Concept getConceptByGpCode(String globalPropertyCode) {
 		return conceptId != null ? Context.getConceptService().getConcept(conceptId) : null;
 	}
 
+	public Boolean getDxfToAdxSwitchConcept() {
+		return "true".equals(Context.getAdministrationService().getGlobalProperty(DXF_TO_ADX_SWITCH));
```

Comment:
DXF_TO_ADX_SWITCH  here should refer to DHISReportingConstants.DXF_TO_ADX_SWITCH i guess where that constant is initialised from